cloud/user-management.md

```diff
@@ -0,0 +1,69 @@
+# User management
+You can manage users for your Timescale Cloud account by navigating to the
+`Members` section in your Cloud console. From here, you can see the current
+members of your Timescale Cloud project, and add and remove members.
```

Comment:
You can use the`Members` section in your Cloud console to view, remove, or add members to your Timescale Cloud project.